src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1967,6 +1967,12 @@ public static IEnumerable<DValue<RecordValue>> StandardTableNodeRecords(IRContex
                 else if (!forceSingleColumn && arg is BlankValue bv && tableType.FieldNames.Count() > 1)
```

Comment:
>tableType.FieldNames.Count() > 1 [](http://example.com/codeflow?start=71&length=32)

Do we need this restriction at all? For example, today this throws an exception: `ForAll([1,2,3], If(Value <> 2, {a:Value}))`

Reply:
I would need to investigate this situation as part of another issue.

Reply:
This is actually the same issue - blank values that are not handled properly.